### DISCUSSION TITLE
Azure ENV with Docker and Nextjs

### DISCUSSION THREAD

**Discussion by anicache08** [vercel/next.js#53895] [Help]
### Summary

Hi Guys!

I have this dilemma, I already setup an Azure ENV and Docker successfully getting it but NextJs can't seem to read the env. I really want to use ENV from Azure as it was the most secured way of handling env and opting out to use env files on our project. Thanks in advance!

**Comment by lahans** [comment]
What have you tried? As reading env variables is supported https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables

As long as you have the env promoted to your container it is also accessible from your code.
